docs/changes/latest.inc

```diff
@@ -32,6 +32,9 @@ Enhancements
 - Organize functional connectivity markers in ``junifer.markers.functional_connectivity``
   (:gh:`107` by `Synchon Mandal`_).
 
+- Add support for nilearn computed masks (`compute_epi_mask`, `compute_brain_mask`, `compute_background_mask`,
```

Comment:
Can we have refs for the functions?

Reply:
Here I meant to add the junifer options, not the nilearn functions.

Reply:
What I meant was a map from `junifer` options to `nilearn` functions.

Reply:
it's in the docs.

Reply:
Okay, all good here.